docs/user-guide/src/bmo/root_device_hints.md

```diff
@@ -0,0 +1,119 @@
+# Root Device Hints
+
+Bare-metal machines often have more than one block device, and in many cases
```

Comment:
One day we will unify how we write bare-metal, baremetal or bare metal, but not today :)

Reply:
I think, and wikipedia seems to be consistent with it in https://en.wikipedia.org/wiki/Bare_machine, we need to use bare-metal as an auxiliary word and "bare metal" as a separate concept.

Reply:
FWIW, it appears in OpenStack, we're officially "Bare Metal" for Ironic. I don't think we use that consistently though.

Reply:
I'll resolve this for now, thanks for taking a look and providing input @dtantsur @jayofdoom